src/main/app/Resources/modules/data/types/fields/modals/parameters/components/parameters.jsx

```diff
@@ -113,6 +169,43 @@ class ParametersModal extends Component {
                 displayed: !this.state.restrictions.locked || this.state.restrictions.lockedEditionOnly
               }
             ]
+          }, {
+            id: 'conditions',
```

Comment:
Should be in the `restrictions` form section with a checkbox to enable/disable it (and hide corresponding fields). See other restrictions options.